fedimintd/src/fedimintd.rs

```diff
@@ -58,9 +61,11 @@ pub struct ServerOpts {
 /// #[tokio::main]
 /// async fn main() -> anyhow::Result<()> {
 ///     Fedimintd::new()?
-///         .attach(WalletGen)
-///         .attach(MintGen)
-///         .attach(LightningGen)
+///         /// use `.with_default_modules()` to avoid having
+///         /// to import these manually
```

Comment:
nit: the comments in the code example should only have two slashes since they aren't doc comments